server/homebrew.api.js

```diff
@@ -81,6 +81,24 @@ const updateBrew = (req, res)=>{
 	HomebrewModel.get({ editId: req.params.id })
 		.then((brew)=>{
 			const updateBrew = excludePropsFromUpdate(req.body);
+
+			if(brew.authors.length > 0 && !(!!req.account && brew.authors.includes(req.account?.username))) {
+				console.warn(`User ${req.account?.username} is not an author on brew ${brew._id}`);
+				res.setHeader('Content-Type', 'application/json');
+				return res.status(403).send(JSON.stringify({ message: `You must be added as an author by the document owner to edit this brew!` }));
```

Comment:
So we throw a 403 when the user tries to visit the edit page, as well as trying to save on that edit page. Do we need both? If they can't even get to the edit page do we need this second check here? Mostly I don't like the code duplication....

Reply:
I think the 403 is necessary in both places because they could have been on the page, lost authorship, then tried to save. 

If the code duplication is a problem I can pull that out into a function?

Reply:
In that case, both retrieving a brew and updating a brew occur via this `homebrew.api` file, so I think it makes sense to emit the 403 from here on GET ing a brew rather than in `app.js`. Both updating and getting can use that function.

Reply:
I'm not seeing how `app.js` currently calls anything in `homebrew.api.js` for retrieving the brew on the edit url -- is there a specific spot you're referring to?

Reply:
Ah you're right. `app.js` directly goes to `homebrewModel.js` rather than via the API....

Ok, I guess we can leave it as is for now since that looks like it will require a bigger rework (I think at some point we want all brew interaction with the database passing through the API for consistency).

I guess if you want to leave it as is and just comment on each that there is an identical function in the other file, or make it into a function either works.

Reply:
Sounds good! I'll pull it into a helper function somewhere.